lib/meta-boxes.php

```diff
@@ -71,9 +71,30 @@ function igv_cmb_metaboxes() {
   ) );
 
   $exhibitor_metabox->add_field( array(
-    'name'     => __( 'Address', 'cmb2' ),
+    'name'     => __( 'City (English)', 'cmb2' ),
     'desc'     => __( '', 'cmb2' ),
-    'id'       => $prefix . 'exhibitor_address',
+    'id'       => $prefix . 'exhibitor_city_en',
+    'type'     => 'text',
+  ) );
+
+  $exhibitor_metabox->add_field( array(
+    'name'     => __( 'City (Español)', 'cmb2' ),
```

Comment:
this one says it works on theme options https://github.com/jmarceli/integration-cmb2-qtranslate